integration/container_volume_test.go

```diff
@@ -0,0 +1,170 @@
+// +build linux
```

Comment:
The CRI integration test suite is currently Linux-only, and this is testing for regression in a problem observed only on Windows.

Have you been able to run this test on Windows? #5163 might help, although it doesn't currently appear to enable the test suite for Windows on CI yet.

Reply:
Indeed, it is. :) It's a bit of a chicken and egg problem with these commits. :) Ofcourse, when one of them merges, I'll have to make changes to the others too to keep them in line with reality. Getting the Windows PR would help a lot. :)